frontend/src/reducer/currentWorkspace.js

```diff
@@ -169,36 +168,6 @@ export default function currentWorkspace (state = defaultWorkspace, action) {
         recentActivityList: state.recentActivityList.filter(c => !action.workspaceContentList.some(cc => c.id === cc.content_id))
       }
 
-    case `${ADD}/${WORKSPACE_CONTENT_SHARE_FOLDER}`:
```

Comment:
why removing all this ?
shouldn't the sharefolder contents appears in the recent activity list of the dashboard ?

Reply:
Yes, it shouldn't have been removed.